Units/simple-markdown.d/input.md

```diff
@@ -28,6 +28,13 @@
 ##### r #
 ##### s ######
 
+'''sh
```

Comment:
GFM recognizes not `''' ... '''`, but `｀｀｀ ... ｀｀｀` or <code>~~~ ... ~~~</code> as the [Fenced Code Blocks][fcb]. Correct this only, at least.

[fcb]: https://github.github.com/gfm/#fenced-code-blocks

Reply:
Thank you. I will update my change.